docs/en/Sorting/MergeSort.md

```diff
@@ -0,0 +1,28 @@
+# Merge Sort
```

Comment:
Could you rename the file's name to `Merge-Sort.md`, after that add it on [README.md](https://github.com/TawfikYasser/DSA/tree/main/docs/en)